src/components/specialized/Achievements/Chain/Confirm/index.js

```diff
@@ -69,7 +69,7 @@ class AchievementConfirm extends Component {
           className={className}
           data-qa-id={`achievement-${idx}-confirm-button`}
           key='achievement-confirm-button'
-          disabled={!canPerformActions || hadConfirmedAlready}
+          disabled={hadConfirmedAlready}
```

Comment:
I think you must at least have primaryWalletReady => otherwise someone can confirm without having primary wallet ready.
Indeed `canPerformActions` was checking that (you can check in src/components/specialized/Achievements/Chain/Confirm/container.js)
- all Wallets were loaded/recovered (maybe only primary is needed)
- Login was done (ok to remove)